extensions/apps/extensions/src/components/pages/extension-edit-published-page/edit-published-extension-page.tsx

```diff
@@ -20,6 +20,16 @@ import { MAX_GALLERY_IMAGES } from '../../../constants';
 import { AtomContext, formDefaultData } from './main-page';
 import { useAtom } from 'jotai';
 
+const getImageObject = imageWithExtraProps => {
```

Comment:
can you add typings to the `imageWithExtraProps` param? because i don't understand why do you need this function

Reply:
when the image data is saved from the model to the form it contains also the __typename property from graphql, this is to clean that up before performing the mutation, I can add the typings for the image object so that its clear that what is passed here should contain those